org.tango.pogo.pogo_gui/src/org/tango/pogo/pogo_gui/DeviceClass.java

```diff
@@ -593,4 +593,4 @@ public static void main(String[] args) {
 
     //===============================================================
     //===============================================================
-}
+}
```

Comment:
Would be better to keep the newline at the end of the file.  Many editors are configured that way.